### PR TITLE
Show installed drivers versions

### LINKED ISSUE #158: Add installed drivers versions info
Now we know bblfsh server version, but it is not enough. There is no way to figure out installed drivers versions right now. It is a good idea to add it.

### PR DESCRIPTION
**Pull request by carlosms**
Fix #158.

a

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -24,17 +36,42 @@ export function Versions({ loading, error, bblfshdVersion, webClientVersion }) {
...
+
   return (
-    <span>
-      Babelfish server: {bblfshdVersion}. Web client: {webClientVersion}
-    </span>
+    <div>
+      <PaddedRight>
+        Babelfish server: {bblfshdVersion}. Web client: {webClientVersion}
+      </PaddedRight>{' '}
+      <DriversVersions data-tip="hello world">
```
Comment (smacker) [inline src/components/Footer.js]: `hello world`? :)

**Comment by smacker** [comment]
CI is failing

File: src/components/Footer.js [hunk 2]
```diff
@@ -24,17 +36,42 @@ export function Versions({ loading, error, bblfshdVersion, webClientVersion }) {
     return <span>{error}</span>;
   }
 
+  const languageVersions = Object.keys(languages).reduce((acc, k) => {
+    const { version, name } = languages[k];
+
+    if (version === undefined) {
```
Comment (se7entyse7en) [inline src/components/Footer.js]: When this happens?

File: src/components/Footer.js (hunk 1 above)
Comment (carlosms) [reply src/components/Footer.js]: Fixed and rebased, we'll pretend this didn't happen :joy:

File: src/components/Footer.js (hunk 2 above)
Comment (carlosms) [reply src/components/Footer.js]: When the `(auto)` language is found. I changed this to make it more clear in 8d31e67.

**Pull request merged**